addons/phantom_camera/scripts/phantom_camera/phantom_camera_properties.gd

```diff
@@ -205,6 +206,12 @@ func add_follow_properties() -> Array:
 				"hint_string": "0.01, 100, 0.01,",
 				"usage": PROPERTY_USAGE_DEFAULT,
 			})
+	_property_list.append({
+		"name": Constants.FOLLOW_PIXEL_PERFECT_PROPERTY_NAME,
+		"type": TYPE_BOOL,
+		"hint": PROPERTY_HINT_NONE,
+		"usage": PROPERTY_USAGE_DEFAULT,
+	})
```

Comment:
Likewise, should be moved to sit below line 94 in `PhantomCamera2D.gd` between the two `append_array()` calls.

```gdscript
property_list.append_array(Properties.add_follow_properties())

_property_list.append({
    "name": Constants.FOLLOW_PIXEL_PERFECT_PROPERTY_NAME,
    "type": TYPE_BOOL,
    "hint": PROPERTY_HINT_NONE,
    "usage": PROPERTY_USAGE_DEFAULT,
})

property_list.append_array(Properties.add_follow_framed())
```